tools/configure.py

```diff
@@ -20,21 +22,63 @@ def register(command):
         default=False,
         help="perform an interactive configuration using ccmake",
     )
+
+    command.add_argument(
+        "-r",
+        "--enable-roles",
+        action="store",
+        dest="enabled_roles",
+        default=None,
+        help="Colon-separated list of roles to enable. Pass 'all' to enable all.",
+    )
+
+    command.add_argument(
+        "-R",
+        "--disable-roles",
+        action="store",
+        dest="disabled_roles",
+        default=None,
+        help="Colon-separated list of roles to disable. If '-R' is set, '-r' is ignored",
+    )
+
```

Comment:
Add a command to list all known roles to make it easier to list enabled/disabled roles
```suggestion
    command.add_argument("-l", "--list-roles", action="store_true", dest="list_roles", default=False, help="List all known roles in the system")
```